<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chan</class>
 <widget class="QWidget" name="chan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>120</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
   </property>
   <property name="text">
    <string>新数据</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>121</width>
     <height>259</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>汉仪秦川飞影W</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
            </property>
            <property name="text">
             <string>姓名</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>汉仪秦川飞影W</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
            </property>
            <property name="text">
             <string>性别</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
          </property>
          <property name="text">
           <string>联系电话</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
          </property>
          <property name="text">
           <string>系</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
          </property>
          <property name="text">
           <string>学历</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>汉仪秦川飞影W</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
   color:rgb(0, 0, 0)
 
}</string>
          </property>
          <property name="text">
           <string>学院</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
   color:rgb(255, 255, 255)
 
}</string>
   </property>
   <property name="text">
    <string>工号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputnumlineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪秦川飞影W</family>
    </font>
   </property>
   <property name="text">
    <string>请输入工号</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>144</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="tellineEdit2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="deplineEdit2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="deglineEdit2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="collegelineEdit2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>144</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="namelineEdit2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="sexlineEdit2"/>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>-10</y>
     <width>1061</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
    background-image:url(:/new/1/others/backgroundimag.jpg);

}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪秦川飞影W</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>确认更改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>600</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>汉仪秦川飞影W</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_15</zorder>
  <zorder>inputnumlineEdit</zorder>
  <zorder>layoutWidget1</zorder>
  <zorder>layoutWidget2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
